components/core/tests/test_log_files/test_no_floats_sorted.json

```diff
@@ -0,0 +1,4 @@
+{"clp_string":"uid=0, CPU usage:99.99%, \"user_name\"=YScope","empty_array":[],"empty_object":{},"false":false,"int16_max":32767,"int16_min":-32768,"int32_max":2147483647,"int32_min":-2147483648,"int64_max_supported":9223372036854776,"int64_min_supported":-9223372036854776,"int8_max":127,"int8_min":-128,"null":null,"string":"short_string","true":true}
```

Comment:
Can you mention in the PR description why int64_max/min_supported is less than 2^63?